ui/pages/remove-snap-account/snap-account-card.tsx

```diff
@@ -24,7 +25,7 @@ export const SnapAccountCard = ({
   const account = mergedAccounts.find(
     (internalAccount: { address: string }) =>
       internalAccount.address === address,
-  );
+  ) as MergedInternalAccount;
```

Comment:
If we show the removal of a snap account, it would mean the account is defined.

Reply:
What would happen if the account was removed in another window while this was open?

Reply:
There might still be a crash here, but probably it would have a minimal impact because it doesn't really make sense to remove an account that doesn't exist. I'll approve for now, but it might be a good idea to follow up here to improve how this missing account scenario is handled